README.md

```diff
@@ -823,22 +823,30 @@ When transferring TOML files over the internet, the appropriate MIME type is
 Comparison with Other Formats
 -----------------------------
 
-In some ways TOML is very similar to JSON: simple, well-specified, and
-maps easily to ubiquitous data types. JSON is great for serializing
-data that will mostly be read and written by computer programs. Where
-TOML differs from JSON is its emphasis on being easy for humans to
-read and write. Comments are a good example: they serve no purpose
-when data is being sent from one program to another, but are very
-helpful in a configuration file that may be edited by hand.
-
-The YAML format is oriented towards configuration files just like
-TOML. For many purposes, however, YAML is an overly complex
-solution. TOML aims for simplicity, a goal which is not apparent in
-the YAML specification: http://www.yaml.org/spec/1.2/spec.html
-
-The INI format is also frequently used for configuration files. The
-format is not standardized, however, and usually does not handle more
-than one or two levels of nesting.
+TOML shares traits with other file formats used for application configuration
+and data serialization, such as YAML and JSON. TOML and JSON both are simple and
+use ubiquitous data types, making them easy to code for or parse with machines.
+TOML and YAML both emphasize human readability features, like comments that make
+it easier to understand the purpose of a given line. TOML differs in combining
+these, rather than permitting the much more complex syntax available in YAML or
```

Comment:
Suggested rephrasing:

> by permitting comments (unlike JSON) and not permitting the much more complex syntax available (unlike YAML).

I'd prefer to leave TOML-and-nesting unaddressed, for right now. It's an open question that we've not come around to figuring out the answer to yet.

Reply:
Reasonable.

Reply:
669501f29281ea04df4a00e217bca902b76d1429 should resolve.